src/Select.js

```diff
@@ -145,6 +145,16 @@ class Select extends Component {
 		return items.map((item) => Soy.toIncDom(item));
 	}
 
+	/**
+	 * Detect changes from the disable attribute.
+	 * @param {string} newValue
+	 */
+	syncDisabled(newValue) {
+		if (newValue) {
+			this.getDropdown().expanded = false;
```

Comment:
Instead of doing this in `syncDisabled`, it'd be best to pass this value via soy, so that the communication of new state is always done there.